azkaban-common/src/main/java/azkaban/executor/container/ContainerizedDispatchManager.java

```diff
@@ -336,6 +350,10 @@ public void run() {
           dsFlow.setStatus(Status.READY);
           dsFlow.setUpdateTime(System.currentTimeMillis());
           ContainerizedDispatchManager.this.executorLoader.updateExecutableFlow(dsFlow);
+          // Emit ready flow event
+          PodEventListener podEventListener = new PodEventListener();
```

Comment:
Use fireEventListeners

Reply:
Used eventhandler to fire event listener.